sapi.services.yml

```diff
@@ -0,0 +1,4 @@
+services:
+  plugin.manager.statisticsplugin.processor:
```

Comment:
this service doesn't include the name of the module that provides it.  Is that acceptable?